frontend/packages/dev-console/src/components/import/git/GitSection.tsx

```diff
@@ -315,6 +315,7 @@ const GitSection: React.FC<GitSectionProps> = ({
           recommendedStrategy: null,
           showEditImportStrategy: true,
         });
+        setFieldValue('build.strategy', BuildStrategyType.Source);
```

Comment:
In which case is this needed? The form jumps already on an URL change to the Builder Image (Source) tab, or? But I'm okay with it as I don't see an issue here.

Generally, the form has issues to keep the state that the user has manually touched.

Reply:
The above change is for below case

1. Switch to developer perspective, add page > import from Git
2. Enter URL https://mysupersecretgit.example.com/org/repo
3. Select Builder image (node.js for example), enter a name. Now all required fields are filled.

Actual results:
The user can not import the Deployment

Before :


https://user-images.githubusercontent.com/102503482/215422237-2c46437e-62bc-4827-b709-47d57691cb1d.mov

After:

https://user-images.githubusercontent.com/102503482/215422281-1e53b496-fc50-4d31-bf20-e29e751d4b09.mov

Reply:
Works fine, thank :+1: